book/src/running-validator/validator-testnet.md

```diff
@@ -17,70 +17,17 @@ tracks the most stable tagged release.
 
 ### Get Testnet Version
 
-You can submit a JSON-RPC request to see the specific version of the cluster.
+You can submit a JSON-RPC request to see the specific software version of the
+cluster. Use this to specify [the software version to install](validator-software.md).
 
 ```bash
-curl -X POST -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","id":1, "method":"getVersion"}' edge.testnet.solana.com:8899
-{"jsonrpc":"2.0","result":{"solana-core":"0.21.0"},"id":1}
+curl -X POST -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0","id":1, "method":"getVersion"}' testnet.solana.com:8899
```

Comment:
Why not use `solana cluster-version` here?

Reply:
Because the user hasn't necessarily installed the software yet